From f22a14a9b64343c56672d30900c3e917028358eb Mon Sep 17 00:00:00 2001
From: Dave Eddy <dave@daveeddy.com>
Date: Thu, 29 Mar 2018 13:14:10 -0400
Subject: [PATCH] TRITON-279 vmapi vms.full.test find_fabric_vm_package
 implemented incorrectly and not needed Reviewed by: Todd Whiteman
 <todd.whiteman@joyent.com> Approved by: Todd Whiteman
 <todd.whiteman@joyent.com>

---
 test/vms.full.test.js | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/test/vms.full.test.js b/test/vms.full.test.js
index 04fcf87..46be1f4 100644
--- a/test/vms.full.test.js
+++ b/test/vms.full.test.js
@@ -2740,16 +2740,6 @@ exports.ensure_no_fabric_nat_provisioned = function (t) {
 };
 
 
-exports.find_fabric_vm_package = function (t) {
-    fabricNetwork = NETWORKS.find(function _findFabricNetwork(n) {
-        return n.fabric === true;
-    });
-
-    t.ok(fabricNetwork, 'found a fabric network');
-    t.done();
-};
-
-
 exports.create_vm_on_fabric_network = function (t) {
     if (!fabricNetwork) {
         t.fail('No fabric network available');
-- 
2.21.0

